### ISSUE TITLE
Expose the possibility to change the log level through the API

### ISSUE THREAD

**Issue by fgalan** [telefonicaid/fiware-orion#1913]
API to implement:

```
POST /admin/log?level=info
```

The value of the "level" parameter is interpreted in case insensitive way and the valid values are:

FATAL, ERROR, WARNING, INFO, DEBUG

Result in the happy case:

```
200 OK (without payload)
```

Result in the case of error:

```
400 Bad Request

{
   "error": "not supported log level"
}
```

**Comment by fgalan** [comment]
Review if all

FATAL, ERROR, WARNING, INFO, DEBUG

make sense for CB. Not sure, but maybe we are not using FATAL

**Comment by fgalan** [comment]
It is under discussion if the verb should be POST or PUT. Let's use POST by the moment (the change would be easy to do if at the end PUT is chosen).

**Comment by kzangeli** [comment]
We might need another error also:

If POST /admin/log is used *without* the URI parameter **level**, I would propose the following response:

    400 Bad Request

    {
      "error": "URI param /level/ missing or empty"
    }

And if there and non-empty but invalid:

    400 Bad Request

    {
      "error": "invalid log level"
    }

**Comment by fgalan** [comment]
Implemented in PR #1919. Moving to QATestPending to be closed by @iariasleon after testing. Problably in this case just manual testing is ok, if this kind of test (which may need inspecting log traces) is difficult to achieve with the python/behave framework.

**Comment by iariasleon** [comment]
issues associated: #2419 and #2420
